### ISSUE TITLE
Web Components

### ISSUE THREAD

**Issue by lastmjs** [ConsenSys/rimble-ui#103]
Wouldn't it be better to create web components that could be easily shared across nearly all projects using any framework or library? I probably won't be using these components sadly, because I don't use React.

**Comment by zkalman** [comment]
We are very in this project. Our initial research among front-end developers working on dApps indicated an overwhelming majority are using React. We are starting there in an effort to get something valuable in the hands of developers quickly. We will definitely consider using web components as the project progresses.

**Comment by TrevorJTClarke** [comment]
I agree with @lastmjs, take a peak at the ionic & their stencil project. If the components are to be widely adopted, they would have a longer shelf life and greater inclusivity with pure web-component standards. As it stands looks like the implementation is early enough where it would be a good option to explore at very least.
